backend/src/main/java/backend/dto/position/PositionDto.java

```diff
@@ -0,0 +1,14 @@
+package backend.dto.position;
+
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
+public class PositionDto {
+
+    private String name;
+
+    private boolean active;
+}
```

Comment:
Stanowisko powinno mieć test, do którego jest przypięte

Reply:
Tak, zmyliła mnie nazwa innego zdania "Przypięcie stanowiska do testu". To test przypisujemy do stanowiska, ale jest na to osobne zadanie